test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Stubs/ExpressionTreeTests/ExpressionTreeToXmlSerializer.cs

```diff
@@ -63,7 +63,7 @@ internal class ExpressionTreeToXmlSerializer : ExpressionVisitor
         public static String SerializeToString(Expression expression)
         {
             outputText = new StringBuilder();
-            writer = XmlTextWriter.Create(outputText);
+            writer = XmlWriter.Create(outputText);

```

Comment:
>XmlWriter [](start = 21, length = 9)

Is it a bug?

Reply:
XmlTextWriter.Create is not supported in some environments.

---
In reply to: [174972609](https://github.com/OData/odata.net/pull/1109#discussion_r174972609) [](ancestors = 174972609)